examples/connectivity/compare_resting_state_decomposition.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
This file should be called 'plot_compare...' to be run by sphinx-gallery to generate a plot.